Integrations/integration-Fortigate.yml

```diff
@@ -92,8 +92,6 @@ script:
         )
         if res.status_code not in {200}:
             return_error('Error in API call to FortiGate [%d] - %s' % (res.status_code, res.reason))
-        if method.upper() != 'GET':
```

Comment:
why?

Reply:
It caused missing data. POST methods return data as well

Reply:
make sure no one uses the statuscode